EasyReflectometry/sample/assemblies/repeating_multilayer.py

```diff
@@ -97,21 +104,10 @@ def from_pars(
             interface=interface,
         )
 
-    @property
-    def uid(self) -> int:
-        """
-        :return: UID from the borg map
-        """
-        return self._borg.map.convert_id_to_key(self)
-
```

Comment:
Was this method no longer needed or something? Doesn't seem like it has been moved

Reply:
It is already in the base class